src/libponyrt/asio/event.h

```diff
@@ -21,6 +21,18 @@ typedef struct asio_event_t
   uint32_t flags;       /* event filter flags */
   bool noisy;           /* prevents termination? */
   uint64_t nsec;        /* nanoseconds for timers */
+
+#ifdef PLATFORM_IS_LINUX
+  // automagically resubscribe to an event on an FD when another event is
+  // on the same FD is triggered
+  // This is only needed on linux where an event firing on an FD that has
+  // ONESHOT enabled will clear all events for the FD and not only the
+  // event that was fired.
+  bool auto_resub;      /* automagically resubscribe? */
+#endif
+
+  bool readable;        /* is fd readable? */
+  bool writeable;       /* is fd writeable? */
```

Comment:
I don't believe so. The size of `asio_event_t` is already over 32 bytes (but well below 64 bytes) before the new additions (assuming I did the math correctly).